integration_testing/features/delete-account.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Delete account
+
+	Background: 
+		Given I am on the *Settings* page
+			And I am on the *Account* tab

```

Comment:
In Kolibri we usually write just `*Settings > Account*` which often makes even one-line background enough :slightly_smiling_face: